packages/odyssey-ie-11-sandbox/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@okta/odyssey-ie-11-sandbox",
+  "version": "0.9.1",
+  "description": "Testing for IE11",
+  "author": "Okta, Inc.",
+  "homepage": "https://github.com/okta/odyssey#readme",
+  "license": "Apache-2.0",
+  "main": "dist/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/okta/odyssey",
+    "directory": "packages/odyssey-ie-11-sandbox"
+  },
+  "scripts": {
+    "start": "webpack serve  --hot --open --mode development",
+    "build": "webpack --mode production"
+  },
+  "dependencies": {
+    "@okta/odyssey-react": "^0.9.1",
+    "core-js": "^3.21.0",
+    "proxy-polyfill": "^0.3.2",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.16.7",
+    "@babel/preset-env": "^7.16.8",
+    "@babel/preset-react": "^7.16.7",
+    "@babel/preset-typescript": "^7.16.7",
+    "babel-loader": "^8.2.3",
+    "css-loader": "^6.5.1",
+    "html-webpack-plugin": "^5.5.0",
+    "sass-loader": "^12.4.0",
+    "style-loader": "^3.3.1",
+    "webpack": "^5.65.0",
+    "webpack-cli": "^4.9.1",
+    "webpack-dev-server": "^4.7.3"
+  },
+  "browserslist": [
+    "ie 11"
+  ]
```

Comment:
we should be able to use our pre-existing browserslist configuration for this!
`extends @okta/browserslist-config-odyssey`

and via env vars:
 
`BROWSERSLIST_ENV=ie11`

Reply:
resolved in 522c43f